generate_composefile.py

```diff
@@ -81,7 +81,7 @@ def parse_config(self, skill_config):
         self.template['build']['args']['skillport'] = skill_config['port']
         self.template['build']['args']['skillconfig'] = skill_config['path']
         self.template['build']['args']['skill_endpoint'] = skill_config['endpoint']
-        self.template['build']['args']['skillhost'] = skill_config['host']
+        self.template['build']['args']['skillhost'] = '0.0.0.0'
```

Comment:
Maybe it's better to create a constant variable in `config.py`. Something like `FLASK_DEFAULT_HOST=="0.0.0.0"`, and then use it everywhere including `skill_server()`.